cubes/server/browser.py

```diff
@@ -18,15 +20,15 @@ def __init__(self, cube, store, locale=None, **options):
         self.locale = locale
         self.store = store
 
-    def features(self):
+    def features(self) -> BrowserFeatures:
 
         # Get the original features as provided by the Slicer server.
         # They are stored in browser_options in the Slicer model provider's
         # cube().
-        features = dict(self.cube.browser_options.get("features", {}))
+        features = BrowserFeatures(**dict(self.cube.browser_options.get("features", {})))
```

Comment:
The initialized should be expanded and each property passed explicitly. We also need that for the enum.